docs/playground/README.md

```diff
@@ -0,0 +1,90 @@
+
+### Wallet pane
+
+The wallet pane has two major parts: (1) a list of wallets with funds and including controls to trigger contract endpoints (i.e., wallet endpoint functions) and (2) a list of the current sequence of wallet events that the user wants to submit for execution. The list of wallets contains at least one and at most 10 wallets. It is accompanied with a "+' button to add additional wallets (greyed out once 10 wallets are displayed). Moreover, the wallet pane contains a "Replay Events" button. It triggers execution of the event sequence on the mockchain.
+
+Each wallet includes the following components:
+
+* **Wallet name:** a string that can be edited by the user;
+* **Wallet funds:** amount in ADA that can be edited by the user (this is the amount at genesis; i.e., start of the emulation); and
+* **Contract endpoints:** These are zero to ten labelled buttons. The labels in all wallets are the same and they are the function names of the contract endpoints defined in the current contract.
+
+The list of current wallet events is an unbound, possibly empty list of sequentially linked events. Each event includes the following components:
+
+* **Event name:** the name of the contract endpoint associated with this event;
+* **Wallet name:** the wallet that this event originated from;
+* **Close button:** clicking this button deletes the corresponding event from the list; and 
+* **Event arguments:** zero to ten key-value pairs. The keys are just string labels, but the values may be structured. The minimal requirement is that the values are multiline strings. Optionally, we render them as either as a string or a nested list of key-value pairs (depending on the type of the corresponding argument of the contract endpoint).
+
+When the user clicks a contract endpoint button in a wallet, a corresponding event is appended to the sequence of wallet events. The wallet name is set to the wallet whose button was clicked. The event arguments are initialised to default empty arguments, where appropriate.
+
+Optionally, the individual events in the sequence of wallet events can be reordered by dragging them around.
+
+The wallet pane is only active after the current contract has been compiled successfully. In fact, only in this case is the correct set of wallet endpoints known. Execution of the event sequence may fail when one or more arguments to one or more events is not a legitimate value of the corresponding type of contract endpoint function. In this case, each event with malformed argument(s) ought to be tinted red, while all malformed argument(s) are highlighted.
+
+### Mockchain pane
+
+The mockchain pane has three major parts.
+
+1. A visual representation of the mockchain in the form of the UTxO graph. The UTxO graph consists of nodes that represent individual (valid) transactions and edges that represent (valid) input-output connections.
```

Comment:
There should also be another type of edge that represents unspent outputs. Otherwise, an output will only appear when it is consumed by an input, and it is hard to see what the funds available to each wallet (in my prototype I faked this by adding dummy transactions that consumed the UTXOs - the `utxo` value [here](https://github.com/input-output-hk/plutus/commit/2340295247d8c28be402e33722d4f28dc1eb8f09#diff-e1c5db9c71eb0e6317ef101381e44debR135)). We could do the same here and make the dummy nodes transparent so one can see that the output is not matched by an input.

Reply:
Yes, very good point.